apps/platform-app/src/app/[locale]/(profile)/(components)/UserPhotoUploader/UserPhotoUploader.tsx

```diff
@@ -7,7 +7,6 @@ import { ErrorIcon } from '@gdh/ui-system/icons'
 import { useSession } from 'next-auth/react'
 import { useTranslations } from 'next-intl'
 import React, { useEffect, useState } from 'react'
-import GithubUserPhotoUploader from './GithubUserPhotoUploader'
 import styles from './UserPhotoUploader.module.scss'
```

Comment:
**suggestion**: Consider using a default avatar URL if `session?.user?.avatarUrl` is not available.